deploy/gen-include-tdishr.py

```diff
@@ -0,0 +1,72 @@
+
+import os
+import csv
+import shutil
+
+from subprocess import *
```

Comment:
is this import even used? besides you should avoid the usage of `from x import *` its not nice to IDEs and humans ;)

I suggest to autopep8 all gen-*.py files .

Reply:
Will do! I actually switched it to `import subprocess` in the other ones, I'll update this one to match.